spec/support/capybara_selenium.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+require 'capybara/rspec'
+require 'selenium-webdriver'
+
+Capybara.register_driver(:headless_chrome) do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    chromeOptions: { args: %w(headless disable-gpu) }
```

Comment:
Now that I see this, I wonder if we need to modify the script above.  Can we just add the `disable-setuid-sandbox` option here instead?